api/v1alpha1/datadogagent_default.go

```diff
@@ -21,8 +22,6 @@ import (
 // default values
 const (
 	defaultLogLevel                         string = "INFO"
-	defaultAgentImageTag                    string = "7.28.0"
-	defaultClusterAgentImageTag             string = "1.12.0"
```

Comment:
Related to my other comment, I think the two constants below should belong to the new package created in this PR.

Reply:
yes